.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -0,0 +1,72 @@
+---
+name: Bug report
+about: Report a problem with XCSoar to help us resolve it
+---
+
+<!--
+
+Welcome! - We kindly ask that you:
+
+  1. Fill out the issue template below - not doing so needs a good reason.
+  2. Use the forum if you have a question rather than a bug or feature request.
+
+The forum is at: https://forum.xcsoar.org
+
+Thanks for understanding, and for contributing to the project!
+
+-->
+
+
+XCSoar version
+--------------
+
+<!--
+Which XCSoar version are you using?
```

Comment:
And which version was the latest which didn't have the problem?

Reply:
I think that a lot of less tech-savvy users discovering a problem in their current version (which might be the first version they try), will not, or might even be unable to, try previous versions to answer this. But I'm totally fine with adding a question about it. Those that can answer will, and those that can't will just say "no idea" or similar.

Reply:
That's not what this is about. Some people report "problem in version X", but they don't say that they didn't have the problem in previous version, and that this is a new problem after updating. Others have never used any other version. Telling the whole story in the bug report saves several roundtrips of asking questions, and that saves everybody's time. Just tell everything you know about the problem.

Reply:
I updated this section to have to questions instead, let's see what you think.